### PR TITLE
Add Icon component

### PR DESCRIPTION
**Pull request by rangoo94**
- Handle custom icons from https://github.com/PublicInMotionGmbH/icon-pack
- Handle Material Design icons

### CODE REVIEW

File: packages/icon/scripts/load-material-icons.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+  // Gather icons list by codepoints
+  const iconsList = readCodepoints(codepointsPath)
+  const iconsAmount = iconsList.length
+
+  // Create map for icons with additional metadata
+  const icons = {}
+
+  // Download all icons
+  await utils.download('icons     ', iconsUrl, iconsPath)
```
Comment (michalochman) [inline packages/icon/scripts/load-material-icons.js]: These extra spaces in label argument across this file are only for pretty printing to console?

File: packages/icon/scripts/utils.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+// Build promisified functions
+const mkdir = promisify(fs.mkdirp)
+const writeFile = promisify(fs.writeFile)
+
+/**
+ * Clear line and display new text
+ *
+ * @param {string} str
+ */
+function writeLine (str) {
```
Comment (michalochman) [inline packages/icon/scripts/utils.js]: This function looks more like an implementation of `overwriteLine`. `writeln` functions usually don't clear current output and they add new line characters.

Comment (michalochman) [reply packages/icon/scripts/utils.js]: Also, it looks like it's not used anyway?

**Comment by michalochman** [comment]
Please resolve conflicts before I can merge.

File: packages/icon/scripts/load-material-icons.js (hunk 1 above)
Comment (rangoo94) [reply packages/icon/scripts/load-material-icons.js]: Yes, exactly

File: packages/icon/scripts/utils.js (hunk 2 above)
Comment (rangoo94) [reply packages/icon/scripts/utils.js]: You're right, it's not used now - I will remove it (as well as `readline` dependency)

**Pull request merged**